dialogue-core/src/main/java/com/palantir/dialogue/core/ConcurrencyLimitedChannel.java

```diff
@@ -39,13 +40,20 @@
 final class ConcurrencyLimitedChannel implements LimitedChannel {
     private static final SafeLogger log = SafeLoggerFactory.get(ConcurrencyLimitedChannel.class);
 
+    @VisibleForTesting
+    static final ChannelState.Key<CautiousIncreaseAggressiveDecreaseConcurrencyLimiter> HOST_SPECIFIC_STATE_KEY =
```

Comment:
this is visible for the purposes of `ConcurrencyLimitedChannelTest#testReuseCachedLimiterState_host`, which may not be a particularly useful test. I'm somewhat indifferent, we could delete that test and make this private.